source/content/wordpress-cron.md

```diff
@@ -1,56 +1,108 @@
+In contrast, any command that can be executed on Unix without user intervention can be scheduled as a Cron job.
 
-### How is WP-Cron triggered?
+### Security
 
-The major difference between Cron and WP-Cron is how WP-Cron is triggered. Cron is a system process that runs every minute and looks for tasks to execute. WP-Cron, because it is a web-based system, can only run when [someone visits the site](https://stackoverflow.com/questions/12895706/when-does-wp-cron-php-run-in-wordpress). Therefore, when someone navigates to your WordPress site, WP-Cron checks to see if anything needs to be done. Thanks to the WordPress core developers, it does this in a way that does not adversely affect the performance of your site.
+WP-Cron is a PHP file that is usually located in the root directory of your WordPress installation. The `wp-cron.php` file is subject to DDoS attacks, but generally, `wp-cron.php` is secure. 
 
-### Problems With Low Traffic Sites
+There is only one parameter you can pass in that will affect the script (`doing_wp_cron`). This `$_GET` value is not filtered; it is only used as a flag and not as input for a process or variable. Beyond that, all input is ignored. While no script on a server attached to the Internet is 100% secure, `wp-cron.php` currently does not have any known vulnerabilities or exploits.
 
-Low traffic WordPress sites may experience skipped tasks when this feature is triggered by visitors. If people aren't visiting your site, WP-Cron can't execute. This doesn't mean your page will be slow from previous jobs when someone eventually does visit your site. Regardless of how many tasks WP-Cron has to execute, they are run in the background so your site's performance is not adversely affected.
+## Manage Jobs with Cron
 
-### Problems With High Traffic Sites
+Follow the steps below to schedule jobs with Cron.
 
-If your WordPress powered site is high traffic, you may run into problems with WP-Cron. The main issues that come up are **race conditions** and **long running processes**.
+1. Disable WP-Cron by adding following code to your `wp-config.php` file:
 
-- **Race condition**: When more than one user visits your site and triggers WP-Cron to execute the same task. WP-Cron works hard to eliminate race conditions, but they can still happen, especially on high traffic sites.
-- **Long running process**: Any task that takes longer than the standard 60 seconds to run. Developers can adjust how long a PHP task is allowed to run with the `set_time_limit()` function. If this is set to be longer than the window between tasks, then you can end up with more than one copy of `wp-cron.php` executing.
+    ```php:title=wp-config.php
+    define('DISABLE_WP_CRON', true);
+    ```
+
+ <Alert title="Note" type="info">
+ You can re-enable WP-cron, if needed, by running the following code:
+
+    ```php
+    define('DISABLE_WP_CRON', false);
+    ```
+
+ </Alert>
+
+1. Run the following code to edit or create your own `crontab` file:
```

Comment:
Why are we telling people how to set up their own cron? Is this for users who need cron run more frequently than once an hour? If that's the case, then perhaps the simplest thing to do would be to leave WP-Cron enabled.

We should be clear about why the user is doing this before telling them how. Also, this section launches straight from the wp-config.php configuration to cron configuration without explaining that the user needs their own dedicated service to run cron.

If we do want to document this, there are a couple of free cron services that we might consider linking to.

Reply:
It should also be clear that none of this is necessary if the user is satisfied with once-an-hour cron job execution.

Reply:
Oh, I see that this section is called "Managing Cron Internally", and there is another very similar section called "Managing Cron Externally" that already includes my suggestion to link to free cron services. It's unclear what context "Managing Cron Internally" is supposed to operate in. You can't run `crontab -e` on your Pantheon site's binding.